File: spec/code_samples/Node_request/list@cleanse@{list_id}/get.js
var request = require("request");

var options = { method: 'GET',
  url: 'http://example.com/api/v1/list/cleanse/%7Blist_id%7D',
  qs: { username: 'SOME_STRING_VALUE' },
  headers:
   { authorization: 'Bearer REPLACE_BEARER_TOKEN',
     'content-type': 'application/json' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: spec/code_samples/Python_requests/list@cleanse@{list_id}/get.py
import requests

url = "http://example.com/api/v1/list/cleanse/%7Blist_id%7D"

querystring = {"username":"SOME_STRING_VALUE"}

headers = {
    'content-type': "application/json",
    'authorization': "Bearer REPLACE_BEARER_TOKEN"
    }

response = requests.request("GET", url, headers=headers, params=querystring)

print(response.text)